RocketChat/Rocket.Chat
# [FIX] Don't show agent info in the transcript if the setting is disabled

## Proposed changes
If the parameter `Show agent info` is disabled then to transcript for the user displayed agent's username.

## Issue(s)


## How to test or reproduce
1. Enable an option `Show agent info`
2. Create Livechat
3. Close livechat
4. Send transcript to the user. Agent name will be available for the user in the transcript
5. Disable the option `Show agent info`
6. Repeat step 2-4. Agent name will not be available for the user in the transcript. The user will see `Agent`

## Types of changes




## Checklist
